serhii.trokhymets/Home_work 3/task_1.rb

```diff
@@ -0,0 +1,29 @@
+class EasyNumber
+  attr_accessor :prime_arr, :bad_number
+  attr_reader :max_value
+
+  def initialize (max_value)
+    @max_value = max_value
+    @prime_arr = []
+    @bad_number = []
+  end
+
+  def find_prime_number
+    (1..@max_value).each do |num|
+      if (1..num).select { |d| num % d == 0 } == [1, num]
+        prime_arr.push(num)
+      end
+    end
+  end
+
+  def display_array
+    find_prime_number
+    (1..@prime_arr.size).each do
```

Comment:
Iterators know the number of array objects, so you do not need to pass them the exact figure
```suggestion
    @prime_arr.siz.each do
      ...
```